frontend/src/pages/contains-tracker-warning.module.scss

```diff
@@ -0,0 +1,79 @@
+@import "../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.contains-tracker-container {
+  margin: $layout-sm;
+
+  @media screen and #{$mq-md} {
+    max-width: $content-xl;
+    margin: $layout-md auto;
+    padding: 0 $layout-sm;
+  }
+
+  .details-section {
+    gap: $layout-xs;
+    display: flex;
+    flex-direction: column;
+
+    h1 {
+      font-family: $font-stack-firefox;
+      border-bottom: 1px solid $color-light-gray-30;
+      padding-bottom: $spacing-xs;
```

Comment:
suggestion: this needs some sort of `@include text-title-xs;` (or something similar) to get the line-height and font-size correct. 

I noticed this when the line-heights were different on mobile: 

![image](https://user-images.githubusercontent.com/2692333/235005432-edf441ce-7145-4d33-a014-40ddacec4024.png)